example/src/main/java/org/wordpress/android/fluxc/example/JetpackAIFragment.kt

```diff
@@ -115,7 +116,36 @@ class JetpackAIFragment : StoreSelectingFragment() {
                             val content = result.choices[0].message?.content
                             content?.let {
                                 prependToLog("Jetpack AI Query Processed:\n$content}")
-                            }?:prependToLog("Error post processing - content is null")
+                            } ?: prependToLog("Error post processing - content is null")
+                        }
+
+                        is JetpackAIQueryResponse.Error -> {
+                            prependToLog("Error post processing: ${result.message}")
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    private fun setJetpackAIQueryQuestionButton() {
+        jetpack_ai_query_question.setOnClickListener {
+            siteStore.sites[0].let {
```

Comment:
```suggestion
           selectedSite?.let {
```

This stumped me for a bit why my test site didn't work. Also consider updating other functions in this file that also uses `siteStore.sites[0]`

Reply:
Great catch!!